packages/loader/container-loader/src/container.ts

```diff
@@ -1645,8 +1614,6 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             }
             if (this.firstConnection) {
                 connectionInitiationReason = "InitialConnect";
-            } else if (this.manualReconnectInProgress) {
-                connectionInitiationReason = "ManualReconnect";
```

Comment:
@vladsud -- Do you know if this distinction of "ManualReconnect" was leveraged much in telemetry analysis?  How big a loss is this?  I don't quite get the semantics of it honestly, but wanted to double-check.

Reply:
Probably? It's useful to learn how quickly document reflects latest changes after user comes back to a tab that was inactive for very long time. That was the reason for this property on connection events. I think it's interesting data to have, but with these changes I think responsibility is pushed to consumers, and I'm fine with it. But we should look closer if consumers have everything, they need to be successful here.